contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--3.1.0--3.2.0.sql

```diff
@@ -6,5 +6,31 @@ SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false)
 DROP CAST IF EXISTS(NUMERIC AS sys.BIT);
 CREATE CAST (NUMERIC AS sys.BIT) WITH FUNCTION sys.numeric_bit (NUMERIC) AS IMPLICIT;
 
+CREATE FUNCTION sys.int4varbinarydiv(leftarg int4 , rightarg sys.bbf_varbinary)
+RETURNS int4
+AS 'babelfishpg_common', 'int4varbinary_div'
+LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+
+
+CREATE OPERATOR sys./ (
+    LEFTARG = int4,
+    RIGHTARG = sys.bbf_varbinary,
+    FUNCTION = int4varbinarydiv,
+    COMMUTATOR = /
+);
+
+CREATE FUNCTION sys.varbinaryint4div(leftarg sys.bbf_varbinary , rightarg int4)
+RETURNS int4
+AS 'babelfishpg_common', 'varbinaryint4_div'
+LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+
+
+CREATE OPERATOR sys./ (
+    LEFTARG = sys.bbf_varbinary,
+    RIGHTARG = int4,
+    FUNCTION = varbinaryint4div,
```

Comment:
which function/operator would be hit if we do not introduce these new operators?

Reply:
Datum int48div(PG_FUNCTION_ARGS)  (https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish/blob/BABEL_3_X_DEV__PG_15_X/src/backend/utils/adt/int8.c#L1014)